content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -0,0 +1,109 @@
+
+Sometimes, customers receive and/or pay their invoice before deciding to return an item within your
+company's return window. In these cases, a return using only *Reverse Transfers* is impossible
+since validated or sent invoices cannot be changed, but *Reverse Transfers* can be used in
+conjunction with *Credit Notes* to complete the customer's return.
+
+Navigate to the customer's :guilabel:`Sales Order`. If a :guilabel:`Sales Order` has any registered
+payment on it, the payment details are located in the chatter, and the invoice itself has a
+:guilabel:`Paid` banner across it.
+
+A customer wants to return 10 of the 20 products that they purchased, already paid for, and
+received. Starting a return in this case begins by clicking on the
+:guilabel:`Validated Delivery Order`, then on :guilabel:`Return` to open the
+:guilabel:`Reverse Transfer` pop-up. Edit the quantity from 20 to 10 since the customer only wants
+to return half of the order, then click :guilabel:`Return`. Just as in Case 1, this action
+generates a new :guilabel:`Warehouse Operation` for the incoming returned product, which can then
+be validated once received. Back on the :guilabel:`Sales Order`, if :guilabel:`Update Quantities`
+was selected, the return product's line item now reflects the change in amount under delivered
+quantities.
+
+.. image:: returns/case-2-updated-sales-quantities.png
+   :align: center
+   :alt: Updated Delivered Quantities
+
+Since the items have already been paid for, the validated invoice needs to be changed to reflect
+that your team processed a return. Tap on the :guilabel:`i` icon next to the paid line item, and
+the :guilabel:`Payment Info` pop-up appears and provides the option to unreconcile this payment.
+Click on :guilabel:`Unreconcile`.
+
+.. image:: returns/unreconcile-button.png
+   :align: center
+   :alt: Unreconcile Button
+
+After the invoice is unreconciled, the options for :guilabel:`Send & Print` and
+:guilabel:`Register Payment` become available again alongside a note that there are outstanding
+payments for the customer.
+
+.. note::
+   The Accounting application needs to be installed for the :guilabel:`Unreconcile` option to be
+   available on an invoice.
+
+Now, select :guilabel:`Add Credit Note`, and a pop-up menu appears where several selections need to
+be made.
+
+.. image:: returns/credit-note-popup.png
+   :align: center
+   :alt: Credit Note pop-up
+
+Choose whether you want the :guilabel:`Credit Method` to be issued as a :guilabel:`Partial Refund`,
+:guilabel:`Full Refund`, or a :guilabel:`Full Refund` and new draft invoice. A reason for the
+credit is not required, but adding one can ensure team members know exactly why a customer was
+credited if looking at orders later. Choosing a :guilabel:`Specific Reversal Date` requires a
+:guilabel:`Refund Date` to also be selected. For both options, a :guilabel:`Specific Journal` is
+not required but can be a helpful tool for team organization.
```

Comment:
Editing this whole paragraph for wordiness, deleting "you", adding important info
```suggestion
Choose whether to issue a :guilabel:`Partial Refund`, :guilabel:`Full Refund`, or :guilabel:`Full Refund and new draft invoice`. The :guilabel:`Partial Refund` option creates a draft Credit Note that can be edited before posting. The :guilabel:`Full Refund and new draft invoice` option validates the Credit Note and duplicates the original invoice as a new draft.

A :guilabel:`Reason` for the credit and a :guilabel:`Specific Journal` to use to process the credit can also be specified. If a :guilabel:`Specific Reversal Date` is selected, then a :guilabel:`Refund Date` must also be selected.
```